docs/reference/setup/install/windows.asciidoc

```diff
@@ -25,236 +28,408 @@ endif::[]
 --------------------------------------------
 
-NOTE: Values that contain spaces must be surrounded with quotes.  For instance `-Epath.logs="C:\My Logs\logs"`.
+By default, msiexec does not wait for the installation process to complete, since it runs in the 
+Windows subsystem. To wait on the process to finish and ensure that `%ERRORLEVEL%` is set
+accordingly, it is recommended to use `start /wait` to create a process and wait for it to exit
 
-TIP: Typically, any cluster-wide settings (like `cluster.name`) should be
-added to the `elasticsearch.yml` config file, while any node-specific settings
-such as `node.name` could be specified on the command line.
+["source","sh",subs="attributes,callouts"]
+--------------------------------------------
+start /wait msiexec.exe /i elasticsearch-{version}.msi /qn
+--------------------------------------------
 
-include::check-running.asciidoc[]
+As with any MSI installation package, a log file for the installation process can be found 
+within the `%TEMP%` directory, with a randomly generated name adhering to the format 
+`MSI*.LOG`. The path to a log file can be supplied using the `/l` command line argument
 
-[[windows-service]]
-==== Installing Elasticsearch as a Service on Windows
+["source","sh",subs="attributes,callouts"]
+--------------------------------------------
+start /wait msiexec.exe /i elasticsearch-{version}.msi /qn /l install.log
+--------------------------------------------
 
-Elasticsearch can be installed as a service to run in the background or start
-automatically at boot time without any user interaction. This can be achieved
-through the `elasticsearch-service.bat` script in the `bin\` folder which allows one to
-install, remove, manage or configure the service and potentially start and
-stop the service, all from the command-line.
+Supported Windows Installer command line arguments can be viewed using
 
 ["source","sh",subs="attributes,callouts"]
---------------------------------------------------
-c:\elasticsearch-{version}{backslash}bin>elasticsearch-service
+--------------------------------------------
+msiexec.exe /help
+--------------------------------------------
 
-Usage: elasticsearch-service.bat install|remove|start|stop|manager [SERVICE_ID]
---------------------------------------------------
+or by consulting the https://msdn.microsoft.com/en-us/library/windows/desktop/aa367988(v=vs.85).aspx[Windows Installer SDK Command-Line Options].
 
-The script requires one parameter (the command to execute) followed by an
-optional one indicating the service id (useful when installing multiple
-Elasticsearch services).
+[[msi-command-line-options]]
+==== Command line options
 
-The commands available are:
+All settings exposed within the GUI are also available as command line arguments (referred to
+as _properties_ within Windows Installer documentation) that can be passed to msiexec:
 
 [horizontal]
-`install`:: Install Elasticsearch as a service
+`INSTALLDIR`:: 
```

Comment:
This seems to be a repetition of a section above.  Could these two not be combined?